modules/programs/lazydocker.nix

```diff
@@ -0,0 +1,62 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.lazydocker;
+
+  yamlFormat = pkgs.formats.yaml { };
+
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+
+in {
+  meta.maintainers = [ hm.maintainers.hausken ];
```

Comment:
You don't seem to already be in the `modules/lib/maintainers.nix` file so you must add yourself. You don't need to put your real name if you feel uncomfortable with that.

Your GitHub user ID is 13941327:

``` console
$ curl -s https://api.github.com/users/hausken | jq .id
13941327
```

Reply:
Hi, thanks for the input! I'm maintainer in nixpkgs now,  so if i understood this correctly i would just need to change this to `maintainers.hausken` for it to refer to my nixpkgs maintainer entry. 
I have pushed this change now and synced branch with upstream